data/json/monsters/mutant_mammal.json

```diff
@@ -364,5 +445,160 @@
       "SWARMS",
       "WARM"
     ]
+  },
+  {
+    "id": "mon_bigdog_greatestswiss",
+    "type": "MONSTER",
+    "name": { "str_sp": "Greatest Swiss" },
+    "description": "This dog is absolutely massive.  It would be big enough to pull a full-sized car.  It might even be ridable.  However, it does appear to be fiercely protective of the other dogs around it.",
```

Comment:
Like doggo but biggo is a bit meh as mutation flavor - something like it moving clumsily because of the bulk, or the fur stretching/tearing because of the muscles would be in order. I can help throw something together if you want, @ me on discord and we can hash something out.